supernode/daemon/mgr/cdn/file_meta_data.go

```diff
@@ -185,10 +183,11 @@ func (mm *fileMetaDataManager) readPieceMD5s(ctx context.Context, taskID, fileMD
 	if err != nil {
 		return nil, err
 	}
-	if err := json.Unmarshal(bytes, &pieceMD5s); err != nil {
-		return nil, err
-	}
+	pieceMD5s = strings.Split(string(bytes), "\n")
 
+	for _, v := range pieceMD5s {
+		logrus.Infof("success to read piece md5 file: %s", v)
```

Comment:
It's better to log this one line with `taskId` on debug level.

Reply:
In fact, they are some code for test. So sorry for that. I have deleted them. 😢